### DISCUSSION TITLE
General Discussion

### DISCUSSION THREAD

**Discussion by k259** [langgenius/dify#10926] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

I'm using Docker to run Dify and I got the password wrong a few times and it's giving me this error "Too many incorrect password attempts. Please try again later."

Can anyone help me resolve it?

**Comment by edmilsonlani** [comment]
Here I am also facing the same problem. I even created a new stack with a new database, but it didn’t help at all.

**Comment by JetPenguin1224** [comment]
I faced the same issue, and here’s how I resolved it:

The  login restriction is associated with the email address. To fix it, you can try the following:

1. Use this command to reset the registered email address:
   ```bash
   docker exec -it docker-api-1 flask reset-email
   ```

2. Then, reset the password with this command:
   ```bash
   docker exec -it docker-api-1 flask reset-password
   ```

After that, you should be able to log in again without any issues.

**Reply by Halfknow** [reply]
I encountered the same problem, if I reset the account, will all the agents data lose and the whole workspace be empty? Or just reset the email account and password and anything else be fine?

**Reply by JetPenguin1224** [reply]
When I tried it, resetting the email account and password did not affect the data; everything remained intact.

**Reply by skikkh** [reply]
The same situation has arisen. We are running it on Cloud Run, and after performing both of the above-mentioned resets once via command, when I attempted to log in again, the same error message appeared and I was unable to log in.

**Reply by skikkh** [reply]
I tried this on 0.12.1, 0.14.0 version.

**Reply by JetPenguin1224** [reply]
I forgot to mention that you need to register a different email address from the previous one.

**Comment by zhangxianglink** [comment]
today ，Upgrading Docker Compose from version 0.8.4 to 0.12.1, I encountered this issue, and the methods mentioned above can handle it.
